### DISCUSSION TITLE
申请退款，完全无法使用

### DISCUSSION THREAD

**Discussion by steqhc** [getlantern/lantern-forum-cn#999] [General]
10月26日购买了专业版，完全无法使用，联系客服[support@getlantern.org]无人回应，申请退款！账户1104707332@qq.com,支付截图如下
![BNS收据](https://github.com/user-attachments/assets/51093fb7-1440-424c-b744-3e3b5cf5f4dd)

**Comment by MaxMKMax** [comment]
@steqhc 您好. 请问您愿意等我们一下吗? 我们会尽快修复好. 请您先别退款.

**Comment by steqhc** [comment]
别尽快修复了，都3个多星期了，依然无法使用，要求退款

**Comment by MaxMKMax** [comment]
@steqhc 您好. 我再重新帮您提交问题多一次.

**Comment by MaxMKMax** [comment]
@steqhc 您好. 请您试试看和支付系统或者银行退款看可以吗?

**Comment by greySkiesMarsh** [comment]
Hi @steqhc 

Your refund was processed on 10/29/2024.
